spec/unitSpec.js

```diff
@@ -33,16 +33,22 @@ describe("Unit#moveTo",[
   it("should add the unit to the new tile",[
     test_soldier.moveTo(fakeTile2),
     expect(fakeTile2.units).toContain(test_soldier)
-    ]),
+  ]),
   it("should remove it from the old tile",[
-    test_soldier.moveTo(fakeTile2), 
+    test_soldier.moveTo(fakeTile2),
     dont(expect(fakeTile.units).toContain(test_soldier))
-    ]),
+  ]),
   it("shouldn't be able to move to tiles that are not adjacent",[
     test_soldier.moveTo(fakeTile3),
     dont(expect(fakeTile3.units).toContain(test_soldier))
-    ])
+  ]),
+  it("should not move unit if that would exceed unit movement range", [
+    test_soldier.moveTo(fakeTile2),
+    test_soldier.moveTo(fakeTile),
+    test_soldier.moveTo(fakeTile2),
+    expect(fakeTile.units).toContain(test_soldier)
```

Comment:
Interesting. I like this. I may have done 
```
dont(expect(fakeTile2.units).toContain(test_soldier))
```
But then I don't think it matters

Reply:
I think either is fine